service/awsconfig/v5/resource/cloudformation/update_test.go

```diff
@@ -282,7 +283,10 @@ func Test_Resource_Cloudformation_newUpdateChange(t *testing.T) {
 
 	for _, tc := range testCases {
 		t.Run(tc.description, func(t *testing.T) {
-			result, err := newResource.newUpdateChange(context.TODO(), clusterTpo, tc.currentState, tc.desiredState)
+			ctx := updateallowedcontext.NewContext(context.Background(), make(chan struct{}))
+			updateallowedcontext.SetUpdateAllowed(ctx)
+
+			result, err := newResource.newUpdateChange(ctx, clusterTpo, tc.currentState, tc.desiredState)
```

Comment:
As of now the tests are not really reflecting the implementation but succeed anyway. Once we have consensus on the update checks above I will sort the tests and improve this a bit. We should maybe also have separate tests for scaling and updates.